builtin/providers/aws/resource_aws_subnet.go

```diff
@@ -108,35 +111,35 @@ func resourceAwsSubnetRead(d *schema.ResourceData, meta interface{}) error {
 
 	subnet := &resp.Subnets[0]
 
-	d.Set("vpc_id", subnet.VpcId)
+	d.Set("vpc_id", subnet.VPCID)
 	d.Set("availability_zone", subnet.AvailabilityZone)
-	d.Set("cidr_block", subnet.CidrBlock)
-	d.Set("map_public_ip_on_launch", subnet.MapPublicIpOnLaunch)
-	d.Set("tags", tagsToMap(subnet.Tags))
+	d.Set("cidr_block", subnet.CIDRBlock)
+	d.Set("map_public_ip_on_launch", subnet.MapPublicIPOnLaunch)
+	d.Set("tags", tagsToMapSDK(subnet.Tags))
```

Comment:
My concern was that they were getting addresses instead of values, but #1104 addresses this case as well, I think